mono/metadata/threadpool-worker.h

```diff
@@ -21,6 +21,17 @@ mono_threadpool_worker_request (void);
 gboolean
 mono_threadpool_worker_notify_completed (void);
 
+#if ENABLE_NETCORE
+gint64
+mono_threadpool_worker_get_completed_threads_count (void);
+
+gint64
+mono_threadpool_worker_get_pending_threads_count (void);
```

Comment:
why isn't this called `mono_threadpool_worker_get_pending_unmanaged_work_item_count`?